### DISCUSSION TITLE
"Movies" vs "Video" playback

### DISCUSSION THREAD

**Discussion by podworx** [rrirower/DP2SceneGraphWizard#33] [Q&A]
In the DP-version of one of my channels, when a "video" was played, when it got to the end, a countdown timer would appear in the upper right of the screen and then the next "video" in line would auto play.

The version created with this tool, "videos" and "movies" behave the same; an in-between "about screen" appears before hitting the PLAY button to start the video.

I liked the auto-load and countdown time function of the original DP version.  Is that something that could be done?

**Comment by rrirower** [comment]
Possibly.  It would have to be an enhancement and not high on the list of things to do.

**Reply by podworx** [reply]
Well, let me vote for the enhancement! ;)

Otherwise, what's the reason behind a "movie" vs a "video"?

**Reply by rrirower** [reply]
The answer to that is found in the [Roku JSON Feed Spec](https://developer.roku.com/docs/specs/direct-publisher-feed-specs/json-dp-spec.md).  Each one has different guidelines for their usage.  Especially when it comes to ad breaks.

**Reply by ferdiworks** [reply]
@podworx The user of the channel can select to play one video or play all videos, that's how the SDK works! We get an extra screen that has additional information!

**Reply by podworx** [reply]
Where would the user find this?  I'm not seeing that option in the interface.

**Reply by rrirower** [reply]
Play all is available for Series only.

**Comment by podworx** [comment]
Here's another quirk I found. 

With one of my channels, one of the "categories" (top-level section on the main page) uses all "videos" (actually referred to in the feed as "shortFormVideos".  All these videos are displayed twice.  So, instead of 14 videos, I have 28.  

The tag is "vvnweb".  The URL for the feed is here:

https://pdwxroku.com/vvn/roku.json

There is one other "shortFormVideo" w/ a different tag. . .it doesn't duplicate like the others.

Any thoughts on why I'd be seeing double for the "vvnweb" tagged videos?

In another one of my channels, "shortFormVideos" in a "category" are triple displayed.

Here's the feed:

http://pdwxroku.com/theonev2/roku.json

The tag is "laughs".

**Reply by ferdiworks** [reply]
@podworx I think you would have more control of your categories and videos is you used Categories with playlist tags

"categories":[{
"name":"3CTV",
"playlistName":"3CTV",
"order":"manual"},
{"name":"Island Vibez",
"playlistName":"Island Vibez",
"order":"manual"},{"name":"Rhythm & Soul",
"playlistName":"Rhythm & Soul",
"order":"manual"},
{"name":"Kool Kountry",
"playlistName":"Kool Kountry","order":"manual"},
{"name":"LIT",
"playlistName":"LIT",
"order":"manual"}],
"playlists":[{"name":"3CTV",
"itemIds":["97784-22388-4286878"]},
{"name":"Island Vibez",
"itemIds":["97786-22390-4286880"]},
{"name":"Rhythm & Soul",
"itemIds":["97788-22392-4286882"]},
{"name":"Kool Kountry",
"itemIds":["97790-22394-4286884"]},
{"name":"LIT",
"itemIds":["97792-22396-4286888"]}]}

**Reply by rrirower** [reply]
This appears to be a bug.  I'm looking into it.  You don't need to define Playlists if you don't want to.

**Reply by rrirower** [reply]
@podworx Can you tell me what content you're seeing in triples?  I think I've fixed the issue but want to check that case.

**Reply by podworx** [reply]
Yes.  Within the https://pdwxroku.com/vvn/roku.json feed, all the shortFormVideo with the tag "vvnweb" are being doubled.

In another one of my channels, "shortFormVideo" in a "category" are triple displayed.

Here's the feed:

http://pdwxroku.com/theonev2/roku.json

**Reply by rrirower** [reply]
@podworx I need to know the specific titles of your content that are shown in triples.

**Reply by podworx** [reply]
Let's start w/ the double issue since it's more prevalent.

In this feed:

https://pdwxroku.com/vvn/roku.json

These titles are duplicating:

The Just One Project with Andrea Martinez – The Maral Vegas Show #009
What the CLUCK with 2020 Presidential Candidate Marianne Williamson – WTC #010
Raised to be in Showbiz – Shelby Jordan – HSP #090
Why We LOVE Living in Las Vegas – TEL #023
LeBron James vs. Michael Jordan – Trash Talkin’ with Legends #008
Embracing Transitions Within Your Business – N2P #052
Two Chicks, One (Painful) Loss – GCS #020
Talktails #073: Rock of Ages Star, Troy Burgess
The Odds Couple #081
Pub Crawl #112: Low to High – Casino Drink Prices
FSE LIVE #010: What To Do for St. Patrick’s Day
Top of the Food Chain #050: The Key to Perfect BBQ At Home
Vegas SportsDay #117 (04-02) NCAA Championship
Awkward Silence 2.1 #009: Comedian Andrew Dice Clay

For the triple issue, in this feed:

http://pdwxroku.com/theonev2/roku.json

The titles that are tripling are:

Two and a Half Canadians - the D Las Vegas
HNL Expansion Draft - the D Las Vegas
Manneken Pis - The D Las Vegas

**Reply by rrirower** [reply]
Thanks. I'll re-test and let you know. I'm confident that I've found the bug.

**Reply by rrirower** [reply]
The next code update will fix this.
